__tests__/app.test.js

```diff
@@ -34,7 +34,7 @@ describe('/api/articles/:article_id', () => {
       });
     });
 
-    test('400: returns "Invalid article ID" for an invalid article_id', async () => {
+    test('400: returns "Bad request" for an invalid article_id', async () => {
```

Comment:
```suggestion
    test('400: returns "Invalid ID" for an invalid article_id', async () => {
```

400 already means bad request, we can be a bit more helpful with our message